data_engineering/pyspark_tasks/task_1.py

```diff
@@ -0,0 +1,47 @@
+"""This module saves top 100 films according to IMDB ratings(all time,60th,last 10 years)"""
+from pyspark import SparkConf
+from pyspark.sql import SparkSession
+
+
+def get_best_alltime(basics_df, ratings_df):
+    return basics_df.join(ratings_df, basics_df['tconst'] == ratings_df['tconst'], 'inner') \
+        .select(basics_df['tconst'], 'primaryTitle', 'numVotes', 'averageRating', 'startYear') \
+        .where('numVotes>=100000 and titleType="movie"') \
+        .orderBy('averageRating', ascending=False).limit(100)
+
+
+def get_best_60th(basics_df,ratings_df):
+    return basics_df.join(ratings_df, basics_df['tconst'] == ratings_df['tconst'], 'inner') \
+        .select(basics_df['tconst'], 'primaryTitle', 'numVotes', 'averageRating', 'startYear') \
+        .where('numVotes>=100000 and titleType="movie" and startYear between 1960 and 1969') \
```

Comment:
It's better to use programmatic API, instead of SQL parts inside the filter, etc. since python do additional code check when interprets code
e.g.
`.where((f.col('numVotes') >= 100_000) & (f.col('titleType') == 'movie') & f.col('startYear').between(1960, 1969))`